downstreamadapter/dispatcher/event_dispatcher.go

```diff
@@ -359,7 +322,7 @@ func (d *Dispatcher) HandleEvents(dispatcherEvents []DispatcherEvent, wakeCallba
 	return block
 }
 
-func (d *Dispatcher) AddDMLEventsToSink(events []*commonEvent.DMLEvent) {
+func (d *EventDispatcher) AddDMLEventsToSink(events []*commonEvent.DMLEvent) {
```

Comment:
we already have the same function in Basic Dispatcher, why we still need this function here?